README.md

```diff
@@ -1,2 +1,98 @@
-# enumerables
-An implementation of some Ruby built-in methods
+# Enumerables
+
+- [Enumerables](#enumerables)
+  - [Description](#description)
+  - [Getting Started](#getting-started)
+  - [Setup](#setup)
+    - [Clone this repository](#clone-this-repository)
+    - [Move into the directory with](#move-into-the-directory-with)
+    - [Install linter](#install-linter)
+    - [Run linter](#run-linter)
+    - [Auto-correct](#auto-correct)
+  - [Authors ✍️](#authors-️)
+  - [🤝 Contributor](#-contributor)
+  - [Show your support](#show-your-support)
+  - [Acknowledgments](#acknowledgments)
+  - [📝 License](#-license)
+  
+## Description
+We built our version of Ruby's Enumerable module by recreating the all, any and filter methods.
+
+## Getting Started
+
+## Setup
+
+### Clone this repository
+
+```bash
+git clone https://github.com/fobadara/enumerables.git
+```
+### Move into the directory with
+
+```bash
+cd enumerables
+```
+
+### Install linter
+
+```bash
+bundle install
+```
+
+### Run linter
+
+```bash
+rubocop .
+```
+
```

Comment:
- [ ] I would recommend including the steps to run the project `ruby main.rb` in the README file.

Reply:
Thank you. This is a very important suggestion.

Reply:
This issue is well resolved! ✔️